cmd/sbctl/bundle.go

```diff
@@ -80,8 +83,13 @@ var bundleCmd = &cobra.Command{
 	},
 }
 
-func bundleCmdFlags(cmd *cobra.Command) {
-	esp := sbctl.GetESP()
+func bundleCmdFlags(cmd *cobra.Command) error {
+	esp, err := sbctl.GetESP()
+	if err != nil {
```

Comment:
If we fail to auto-detect the `ESP`, then there will be no default value for it when generating bundles.

I think it's the best fallback, since the only other option is to fail completely, and user's won't even be able to see `--help`.

Reply:
Yep, I agree!